### ISSUE TITLE
Align first, then add superscript

### ISSUE THREAD

**Issue by nickhaf** [nickhaf/eatPlot#315]
```r
library(ggplot2)

SEr2s <- data.frame(Species = c("virginica",  "setosa",  "versicolor" ), 
                    xpos = c(1,1,1), ypos = c(1,2,3), 
                    lab = c("4.2^a","bold(-3.4)^a", "1.2^phantom(a)"))

## Idee: Zahl zuerst plotten. superscript dann ran packen.

ggplot()+
  geom_text(data = SEr2s, 
            aes(x = xpos, 
         y = ypos,
         label = lab),
         parse = TRUE,
         hjust = 1
  )
```

**Comment by nickhaf** [comment]
Problem:
1) For alignment we would need the phantom operator of plotmath. However, it does not allow for bold printing of numbers. Therefore we need a two step approach:

a) Plot Numbers with ggtext and align them.
b) Add on the outer boder (depending on alignment and rounding) the sig-column, which consists of superscripts (and phantoms?). The phantoms are probably not really needed here, so we could use either ggtext or geom_text(parse = TRUE).

**Comment by nickhaf** [comment]
closed in #317
